dev/ScrollBar/ScrollBar_themeresources.xaml

```diff
@@ -210,6 +213,7 @@
         <Setter Property="Background" Value="{ThemeResource ScrollBarBackground}" />
         <Setter Property="Foreground" Value="{ThemeResource ScrollBarForeground}" />
         <Setter Property="BorderBrush" Value="{ThemeResource ScrollBarBorderBrush}" />
+        <Setter Property="Margin" Value="{StaticResource ScrollBarMargin}"/>
```

Comment:
nit: I would move the margin setter under MinHeight to group the StaticResources together